scalyr_agent/agent_main.py

```diff
@@ -766,8 +766,8 @@ def __detailed_status(
                     print("Cannot get health check result.")
             elif (
                 status_format == "text"
-                and "Health check" in line
-                and line.rstrip() != "Health check: Good"
+                and "Health check:" in line
+                and "Good" not in line.rstrip()
```

Comment:
This line would alternatively contain the error message, which could in theory contain "Good", seems like a long shot but something that we could trip over without noticing. I think some kind of more strict check is needed here.

Reply:
Yeah, I wanted to keep it simple, but I can add regex which handler both scenarios: ``^Health check\:\s+Good$"``

Reply:
Here we go - 2012790598effa1917007e768f1929618d1641c1.